common/src/test/java/com/microsoft/identity/common/nativeauth/java/util/CommandResultUtilTest.kt

```diff
@@ -20,14 +20,14 @@
 //  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 //  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //  THE SOFTWARE.
-package com.microsoft.identity.common.internal.util
+package com.microsoft.identity.common.java.nativeauth.util
```

Comment:
I believe this one **_doesn't_** need `java`

Reply:
I have updated it

Reply:
Previous comment deleted. I will fix this.